web/auth/forms.py

```diff
@@ -4,6 +4,8 @@
 from wtforms import StringField, PasswordField, SubmitField
 from wtforms.validators import InputRequired, Length, ValidationError
 
+from lib.Config import Configuration
+config = Configuration()
```

Comment:
What's the rationale behind these 2 lines?

Reply:
@P-T-I , I wanted to call the conf methods to retrieved the OIDC conf like what I saw in the admin page but since the conf is already loaded in the run.py, I have removed these lines now and I'm just referring to the global var config

Reply:
Ok; but these two lines do not do anything inside the python document / module they are currently in, right? If so, why are they in here?

Reply:
@P-T-I, I have removed them. Sorry I wasn't looking at the right file...